docs/overview.md

```diff
@@ -33,29 +32,41 @@ code that drives Sprocket is split across the [`wdl`] family of crates, the
 
 ## Project Goals
 
-### High-Performance Orchestration Engine
+### High-Performance Workflow Execution Engine
 
-Most fundamentally, the Sprocket project was created due to the lack of workflow
-engines that were powerful enough to orchestrate tens of thousands or hundreds
-of thousands of concurrent jobs. Enabling the next generation of large-scale,
-open science will always remain the top-level objective for the project.
+Fundamentally, the Sprocket project was created to address the lack of
+bioinformatics workflow execution engines that can reliably handle tens of
+thousands to hundreds of thousands of concurrently executing workflows across a
+variety of execution environments.
```

Comment:
I'm not sure we necessarily need an answer to this, but I would point out that I'm not sure we have an infrastructure capable of running 10k - 100k workflows concurrently. Maybe if we were able to leverage multiple backends.

Reply:
For the public record, this was my response to Andrew's thought during an internal conversation.

> For me, when I say "handling tens of thousands to hundreds of thousands of concurrently executing workflows", I don't necessarily mean that every workflow is executing at every time point. Instead, I'm referring to the experience where one can "submit and forget" hundreds of thousands of workflows to an orchestrator, and those workflows will execute as quickly as possible (similar to common HPC schedulers today). 
>
> In terms of the concurrent execution of workflows within a particular runtime, that's going to be dependent on a ton of factors, and we'll just have to gather metrics to see what is actually possible.